lib/Alien/Build/Interpolate.pm

```diff
@@ -196,12 +204,18 @@ sub _get_prop
   }
 }
 
-# TODO: document $prop
-
 sub interpolate
 {
   my($self, $string, $prop) = @_;
-  $prop ||= {};
+
+  if(eval { $prop->isa('Alien::Build') })
+  {
+    $prop = $prop->_command_prop;
+  }
+  else
+  {
+    $prop ||= {};
```

Comment:
Maybe should rewrite calls to this method so that they use the new form instead of this older one.

Reply:
Fixed in #340